resources/views/layouts/_fathom.blade.php

```diff
@@ -1,5 +1,5 @@
-@production
+@env(['production', 'testing'])
```

Comment:
Hmm we definitely don't want to be doing this. This could be something we'd easily miss some day if we introduce, for example, Dusk tests.

Reply:
That's fair, but I'd like to be able to test. That might be a non-issue if we permanently leave canonical = false